skins/oasis/css/core/ads.scss

```diff
@@ -270,16 +272,88 @@ body >a >img[src$="noad.gif"] {
 
 // Recovered message
 .recovered-message {
-	background: #eee;
-	color: #000;
-	font-size: 2em;
 	text-align: center;
 	padding: 10px 0;
+	min-height: 100px;
+
+	.message {
+		background: #ffffff;
+		color: #000000;
+		padding: 10px 12px;
+		border: 2px solid #000000;
```

Comment:
Please shorten all the color hexes from `#ffffff` to `#fff`, `#000000` to `#000` and so on.

Additional question is: how does it look on light and dark wikis? If it doesn't look good on dark wikis then it makes sense to define variables for the colors i.e.:
```
$message-background-color-light: #000;
$message-background-color-dark: #fff;

$message-text-color-light: #000;
$message-text-color-dark: #fff;
```